builtin/providers/aws/resource_aws_autoscaling_scaling_policy_test.go

```diff
@@ -0,0 +1,125 @@
+package aws
+
+import(
+    "fmt"
+    //"math/rand"
+    //"strings"
+    "testing"
+    //"time"
+
+    "github.com/awslabs/aws-sdk-go/aws"
+    "github.com/awslabs/aws-sdk-go/service/autoscaling"
+    "github.com/hashicorp/terraform/helper/resource"
+    "github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccAWSAutoscalingScalingPolicy_basic(t *testing.T) {
+    var policy autoscaling.ScalingPolicy
```

Comment:
I see, I'm saying it's not _used_. In `testAccCheckScalingPolicyExists`:

```go
*policy = *resp.ScalingPolicies[0]
```

That's the only place `policy` is used, in an assignment. It's never referenced again, and none of its properties are ever checked. 

You can omit the var and line (54) there, unless you'd like to pass `policy` to a separate function and check it's properties like we do in [Security Group](https://github.com/hashicorp/terraform/blob/master/builtin/providers/aws/resource_aws_security_group_test.go#L26-L27)